### ISSUE TITLE
Write manual doc from deploying/ testing

### ISSUE THREAD

**Issue by FBoucher** [FBoucher/TinyBlazorAdmin#4]
- [x] Create AzUrlShortener headless
- [x] Create Azure Active Directory (AAD) components
- [x] Configure the AAD components
- [ ] How to deploy the TinyBlazorAdmin website
- [ ] Configure the backend and the front

**Comment by alsmyr** [comment]
Trying to follow the steps in the current manual doc but I am not successful on closing the loop on the final step. https://github.com/FBoucher/TinyBlazorAdmin/blob/main/deployment.md#configure-backend-and-frontend-to-work-together

Same as one of the unfinished checks in this issue.

I'd be happy to update the doc once I have managed to follow. 

The two keys to enter into appsettings.json; which IDs from the steps above are they? I assued it was the Tenant ID of my AD and the Client ID of the 1st registered app but that did not work for me.
